examples/tutorials/vectors.py

```diff
@@ -0,0 +1,459 @@
+"""
+Plot vectors
+============
+
+Plotting vectors is handled by :meth:`pygmt.Figure.plot`.
+
+.. note::
+
+    This tutorial assumes the use of a Python notebook, such as IPython or Jupyter Notebook.
+    To see the figures while using a Python script instead, use
+    ``fig.show(method="external")`` to display the figure in the default PDF viewer.
+
+    To save the figure, use ``fig.savefig("figname.pdf")`` where ``"figname.pdf"``
+    is the desired name and file extension for the saved figure.
+"""
+# sphinx_gallery_thumbnail_number = 6
+
+import numpy as np
+import pygmt
+
+########################################################################################
+# Plot Caretesian Vectors
+# -----------------------
+#
+# Create a simple Cartesian vector using a starting point through
+# ``x``, ``y``, and ``direction`` parameters. The direction is specified
+# by a list of two 1d arrays structured as ``[[angle_in_degrees], [length]]``
+#
+# On the shown figure, the plot is projected on a 10cm X 10cm region,
+# which is specified by the ``projection`` parameter.
+# The length of the vector also uses centimeters by default but
+# could be changed using :meth:`pygmt.config`
+# (Check the next examples for unit changes).
+#
+# Notice that the ``v`` in the ``style`` parameter stands for
+# vector; it distinguishes it from regular lines and allows for
+# different customization. ``0c`` is used to specify the size
+# of the arrowhead which explains why there is no arrow on either
+# side of the vector.
```

Comment:
See [-Sv](https://docs.generic-mapping-tools.org/latest/plot.html#s) in GMT documentation:
> vector. Direction (in degrees counter-clockwise from horizontal)
```suggestion
# side of the vector. The angle is degrees counter-clockwise from horizontal.
```

Reply:
I just added a similar sentence at the beginning of the paragraph, I thought it works better there.

Reply:
Great~